backend/src/main/java/gov/cdc/usds/simplereport/idp/repository/DemoOktaRepository.java

```diff
@@ -109,7 +120,8 @@ public void reprovisionUser(IdentityAttributes userIdentity) {
       throw new IllegalGraphqlArgumentException(
           "Cannot update Okta user privileges for nonexistent organization.");
     }
-    if (!orgUsernamesMap.get(orgId).contains(username)) {
+    if (!tenantDataContextHolder.hasBeenPopulated()
```

Comment:
what are you trying to ascertain by checking the tenant data access here?

Reply:
Great catch!  This should not have been added, I've also reverted a similar change to `LiveOktaRepository`.